common-mistakes/deb-download-untrusted.rst

```diff
@@ -1,49 +1,53 @@
 Downloading .debs from untrusted sources
 ########################################
 
-While packages offer a convenient method to package and distribute software, 
-this ease of distribution inherently comes with risks. As an example, 
-consider a scenario where a user has provided a direct download link for a 
-package on a public forum. Downloading and installing this package from such
-a source is **discouraged** for several reasons.
+Packages offer a convenient way to distribute software, but this ease of
+distribution comes with risks. For example, a user might provide a direct
+download link for a package on a public forum.
```

Comment:
I want it to remain clear we are stating that public forums are untrusted sources.
```suggestion
download link for a package on a public forum or another untrusted source.
```